kustomize/base/brigadasos-db.yaml

```diff
@@ -24,12 +27,45 @@ metadata:
     app: brigadasos-db
 spec:
   storageClassName: manual
+  volumeName: brigadasos-db-pvolume
   accessModes:
     - ReadWriteMany
   resources:
     requests:
       storage: 50Gi
 ---
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: brigadasos-db-temp-pvolume
+  namespace: nadedb
+  labels:
+    type: local
+    app: brigadasos-db
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 50Gi
```

Comment:
Not really sure why we have to specify the capacity storage twice, but just in case this should also be `20Gi`